src/driver_arguments.cpp

```diff
@@ -66,6 +66,10 @@ void BnDataType(std::stringstream& ss, const miopen::TensorDescriptor& desc)
     {
         ss << "bnormfp16";
     }
+    if(desc.GetType() == miopenBFloat16)
+    {
+        ss << "bnormbfp16";
+    }
```

Comment:
Not sure we have enough details here to know, but do we need to handle "bnormfp16fp32" here as well?

And should this be "bnormbfp16fp32" instead?

Reply:
you are correct! There is not enough details to find out where the run was "bnormfp16" or "bnormfp16fp32" so I defaulted to "bnormfp16"